DOCS-5144 Adds Build an Integration, Build a Marketplace Offering, and Create a Tile Documentation (#14297)
DataDog/integrations-core: docs/dev/marketplace_offering.md

[1]: https://app.datadoghq.com/integrations
[2]: https://app.datadoghq.com/marketplace
[3]: https://partners.datadoghq.com/
[4]: https://docs.datadoghq.com/developers/integrations/new_check_howto/
[5]: https://docs.datadoghq.com/developers/datadog_apps
[6]: https://partners.datadoghq.com/English/
[7]: /account_management/users/#add-new-members-and-manage-invites
[8]: https://learn.datadoghq.com/courses/intro-to-integrations
[9]: https://learn.datadoghq.com/
[10]: https://chat.datadoghq.com/
[11]: https://docs.datadoghq.com/developers/authorization/
[12]: https://github.com/DataDog/marketplace/pull/107
[13]: https://docs.datadoghq.com/developers/integrations/create_a_tile
[14]: https://docs.datadoghq.com/developers/integrations/api_integration/
[15]: https://docs.datadoghq.com/integrations/create_a_tile/#agent-based-integrations
[16]: https://docs.datadoghq.com/integrations/create_a_tile/#rest-api-integrations
[17]: https://docs.datadoghq.com/integrations/create_a_tile/#datadog-apps
[18]: https://docs.datadoghq.com/integrations/create_a_tile/#saas-license-or-professional-service-offerings
[19]: https://docs.datadoghq.com/developers/integrations/agent_integration

> [20]: https://www.dashcon.io/

Author: ```suggestion
[20]: https://www.dashcon.io/
[21]: https://www.datadoghq.com/blog/
```

---
title: Build A Marketplace Offering
type: documentation
aliases: 
- 'developers/marketplace/'

> dependencies: "https://github.com/DataDog/integrations-core/blob/alai97/add-marketplace-documentation/docs/dev/marketplace_offering.md"

Author: ```suggestion
dependencies: "https://github.com/DataDog/integrations-core/blob/master/docs/dev/marketplace_offering.md"
```

{{< img src="developers/marketplace/marketplace_overview.png" alt="The Datadog Marketplace page" style="width:100%" >}}

## List an offering 

The following types of offerings are supported on the Datadog Marketplace:

Integrations
: Marketplace integrations that submit third-party data to (or pull data from) a user's Datadog account through the [Datadog Agent][15] or the [Datadog API][16]. These integrations can contain a variety of data types, such as metrics, events, logs, traces, and more.

Software licenses
: Software licenses enable you to deliver and license software solutions to customers through the Datadog Marketplace.

Professional services
: Professional services enable you to offer your team's services for implementation, support, or management for a set period of time.

## Join the Datadog Marketplace 

Marketplace Partners have unique benefits that are not available to Technology Partners who list out-of-the-box integrations:
 
- **Go-to-market collaboration** including a blog post, a quote for a press release, and social media amplification, with access to dedicated sales and marketing resources focused on accelerating partner growth. 

> - **Training and support** for sales enablement in the form of on-demand courses to quickly onboard sales and technical teams.

Reviewer: This may confuse partners, who could expect to have dedicated training sessions with our sales team, which we are not able to provide. 

Separately, I think we should add reaching Datadog's 25,000+ customers in this benefits section. That number is public in our quarterly earnings report.

Reviewer 2: Changed to "Training and support for internal sales enablement".

@alexvetras on the bit about reaching customers, I don't think thats a benefit that's unique to the marketplace page. Should we instead add that to the general benefits section in the readme.md (overview doc)?

1. Log into the [Datadog Partner Portal][6].
2. On your personal homepage, click on the **Learn More** button under **Sandbox Access**.
3. Select **Request Sandbox Upgrade**.

<div class="alert alert-info">If you are already a member of a Datadog organization (including a trial org), you may need to switch to your newly created sandbox. For more information, see the <a href="https://docs.datadoghq.com/account_management/org_switching/">Account Management documentation</a>.</div>

Creating a developer sandbox may take up to one or two business days. Once your sandbox is created, you can [invite new members from your organization][7] to collaborate with.

## Explore learning resources

Once you've joined the Technology Partners track and requested a sandbox account, you can start learning about developing Datadog integrations and additional offerings by:

* Completing the on-demand [**Introduction to Datadog Integrations**][8] course on the [Datadog Learning Center][9].

> * Joining the Marketplace Engineering Office Hours through the public [Datadog Slack channel][10].

Reviewer 3: @anawishnoff Should we remove the 

>Reading the documentation about [Datadog Apps][5] if you want to build a custom widget that integrates external data and actions with Datadog dashboards.

Integrations
: Marketplace integrations that submit or pull third-party data through the [Datadog Agent][15] or an [API][16]. These integrations contain out-of-the-box metrics, events, or service checks.

Software licenses
: SaaS licenses enable you to deliver and license software solutions to customers through the Datadog Marketplace.

UI extensions or applications
: An application (such as a [Datadog App][5]) with a [tile-only listing][17] on the Datadog Marketplace.

Professional services
: [Professional services][18] enable you to offer your team's services for implementation, support, or management for a set period of time.

### Request access to Marketplace

To request access to the private Marketplace repository, email <a href="mailto:marketplace@datadoghq.com">marketplace@datadoghq.com</a>. A Marketplace offering requires files and information such as pricing plans. Once you have been granted access, you can review an [example pull request][12] in the Marketplace repository with annotations and best practices.

> To get started with creating an offering, see [Create a Tile][13].

Reviewer 2: Can we add a heading above this to make it more prominent?

---
title: Build A Marketplace Offering
type: documentation
aliases: 
- 'developers/marketplace/'
dependencies: "https://github.com/DataDog/integrations-core/blob/alai97/add-marketplace-documentation/docs/dev/marketplace_offering.md"
further_reading:
- link: "https://www.datadoghq.com/partner/"
  tag: "Partner Network"
  text: "Datadog Partner Network"
- link: "https://www.datadoghq.com/blog/datadog-marketplace/"
  tag: "Blog"
  text: "Expand your monitoring reach with the Datadog Marketplace"
- link: "/developers/integrations/marketplace_offering/"
  tag: "Documentation"
  text: "Learn how to develop a Marketplace offering"
- link: "/developers/datadog_apps/"
  tag: "Documentation"
  text: "Learn about Datadog Apps"
---

## Overview

The Datadog Marketplace is a digital marketplace where Datadog Technology Partners can list their paid offerings to Datadog users.  

> While the **Integrations** page includes integrations and custom dashboard widgets built by both Datadog and Technology Partners at no cost, the **Marketplace** page is a commercial platform for Datadog customers and Technology Partners to buy and sell a variety of offerings including Agent-based or API-based integrations, custom dashboard widgets, software subscriptions/licenses, and professional services.

Reviewer 3: license

## List an offering 

All Technology Partners can list an out-of-the-box integration on the **Integrations** page, or a commercial paid offering on the **Marketplace** page. The following types of offerings are supported on the Datadog Marketplace:

### Integrations

> Marketplace integrations that submit third-party data or pull data from a user's Datadog account through the [Datadog Agent][15] or the [Datadog API][16]. These integrations can contain a variety of data such as out-of-the-box metrics, events, logs, and more.

Reviewer: ```suggestion
Marketplace integrations that submit third-party data or pull data from a user's Datadog account through the [Datadog Agent][15] or the [Datadog API][16]. These integrations can contain a variety of data types, such as metrics, events, logs, traces, and more.
```

### Software licenses
Software licenses, or subscriptions to SaaS solutions, enable you to deliver and license software solutions to customers through the Datadog Marketplace.

> ### Custom dashboard widgets

Reviewer 4: Are we proposing these?

A custom dashboard widget that adds visual functionality to existing integration data (or other data within Datadog).

### Professional services
[Professional services][18] enable you to offer your team's services for implementation, support, or management for a set period of time.

> ## Why join the Datadog Marketplace? 

Reviewer 4: Should we call out generating new leads?

Marketplace Partners have unique benefits that are not available to partners who list out-of-the-box integrations:
 
  - Go-to-Market motions, including a blog post, quote for a press release, and social media amplification, with access to dedicated sales and marketing resources, focused on accelerating partner growth. 

  - Training and Support for sales enablement in the form of on-demand courses to quickly onboard sales and technical teams.

  - Exclusive opportunities to sponsor conferences and events (such as Datadog DASH) at a discounted rate.

{{< img src="developers/marketplace/marketplace_overview.png" alt="The Datadog Marketplace page" style="width:100%" >}}

## Join the Datadog partner network

> Before listing an offering on the Datadog Marketplace, you will first need to apply to the [Datadog Partner Network's][3] **Technology Partner** track. Once your application has been approved, you can begin to develop your offering.

Reviewer: When need all partners to join the DPN, not just those developing for the Marketplace. 

In these new docs, if a partner is building for the Integrations page, where are we explaining the join DPN, request sandbox, etc. steps?

> Before listing an offering on the Datadog Marketplace, you will first need to apply to the [Datadog Partner Network's][3] **Technology Partner** track. Once your application has been approved, you can begin to develop your offering.

Reviewer 3: @anawishnoff Should we include the same steps on the landing page for agent or should we change this to support ootb integrations and marketplace

type: documentation
aliases: 
- 'developers/marketplace/'
dependencies: "https://github.com/DataDog/integrations-core/blob/alai97/add-marketplace-documentation/docs/dev/marketplace_offering.md"
further_reading:
- link: "https://www.datadoghq.com/partner/"
  tag: "Partner Network"
  text: "Datadog Partner Network"
- link: "https://www.datadoghq.com/blog/datadog-marketplace/"
  tag: "Blog"
  text: "Expand your monitoring reach with the Datadog Marketplace"
- link: "/developers/integrations/marketplace_offering/"
  tag: "Documentation"
  text: "Learn how to develop a Marketplace offering"
- link: "/developers/datadog_apps/"
  tag: "Documentation"
  text: "Learn about Datadog Apps"
---

## Overview

The Datadog Marketplace is a digital marketplace where Datadog Technology Partners can list their paid offerings to Datadog users.  

While the **Integrations** page includes integrations built by both Datadog and Technology Partners at no cost, the **Marketplace**  is a commercial platform for Datadog customers and Technology Partners to buy and sell a variety of offerings including Agent-based or API-based integrations, software licenses, and professional services.

## List an offering 

> All Technology Partners can list an out-of-the-box integration on the **Integrations** page, or a commercial paid offering on the **Marketplace** page. The following types of offerings are supported on the Datadog Marketplace:

Reviewer 4: I don't think we need to call out the Integrations page here since this is specific to the Marketplace.

### Integrations
Marketplace integrations that submit third-party data or pull data from a user's Datadog account through the [Datadog Agent][15] or the [Datadog API][16]. These integrations can contain a variety of data types, such as metrics, events, logs, traces, and more.

### Software licenses
Software licenses enable you to deliver and license software solutions to customers through the Datadog Marketplace.

### Professional services

> [Professional services][18] enable you to offer your team's services for implementation, support, or management for a set period of time.

Reviewer 4: Is this link right?

## Why join the Datadog Marketplace? 

Marketplace Partners have unique benefits that are not available to partners who list out-of-the-box integrations:
 

>   - Go-to-Market collaboration, including a blog post, a quote for a press release, and social media amplification, with access to dedicated sales and marketing resources focused on accelerating partner growth. 

Reviewer 4: Can we bold this?

  - Training and Support for sales enablement in the form of on-demand courses to quickly onboard sales and technical teams.

  - Exclusive opportunities to sponsor conferences and events (such as Datadog DASH) at a discounted rate.
  
  - Generating new leads from user discovery

{{< img src="developers/marketplace/marketplace_overview.png" alt="The Datadog Marketplace page" style="width:100%" >}}

> ## Join the Datadog partner network

Reviewer 4: Should this be moved up?

Reviewer 2: I think it makes sense here - the flow is that a user will read about what the marketplace is and what they can list there, understand the benefits of the marketplace, and then apply to be a tech partner if all of that sounds intriguing.

Before listing an offering on the Datadog Marketplace, you will first need to apply to the [Datadog Partner Network's][3] **Technology Partner** track. Once your application has been approved, you can begin to develop your offering.

## Request a sandbox account

All Technology Partners can request a dedicated sandbox Datadog account to aid in their development.

To request a sandbox account:

1. Log into the [Datadog Partner Portal][6].
2. On your personal homepage, click on the **Learn More** button under **Sandbox Access**.
3. Select **Request Sandbox Upgrade**.

<div class="alert alert-info">If you are already a member of a Datadog organization (including a trial org), you may need to switch to your newly created sandbox. For more information, see the <a href="https://docs.datadoghq.com/account_management/org_switching/">Account Management documentation</a>.</div>

Creating a developer sandbox may take up to one or two business days. Once your sandbox is created, you can [invite new members from your organization][7] to collaborate with.

> ## Explore learning resources

Reviewer 4: This might be better on the main Integrations development page.

Once you've joined the Technology Partner track and requested a sandbox account, you can start learning about developing offerings by:

* Completing the on-demand [**Introduction to Datadog Integrations**][8] course on the [Datadog Learning Center][9].
* Reading the documentation about setting up an [OAuth 2.0 client][11] for API-based integrations.

### Request access to Marketplace

To request access to the private Marketplace repository, email <a href="mailto:marketplace@datadoghq.com">marketplace@datadoghq.com</a>. Once you have been granted access, you can review an [example pull request][12] in the Marketplace repository with annotations and best practices.

## Coordinate GTM opportunities

Once a Marketplace tile is live, Technology Partners can meet with Datadog's Partner Marketing team to coordinate a joint go-to-market (GTM) strategy, which includes the following:

- A Datadog quote for partner press releases
- A blog post on the [Datadog Monitor][23]
- Amplification of social media posts

## Getting Started

> To get started with creating an offering, see [Create a Tile][13]. If you're interesting in building an Agent-based integration and selling it on the Datadog Marketplace, see [Create an Agent-based Integration][19].

Reviewer 4: Should we also list out building API-based integration?

Reviewer 2: Clarified in upcoming commit

3. Select **Request Sandbox Upgrade**.

<div class="alert alert-info">If you are already a member of a Datadog organization (including a trial org), you may need to switch to your newly created sandbox. For more information, see the <a href="https://docs.datadoghq.com/account_management/org_switching/">Account Management documentation</a>.</div>

Creating a developer sandbox may take up to one or two business days. Once your sandbox is created, you can [invite new members from your organization][7] to collaborate with.

## Request access to Marketplace

To request access to the private Marketplace repository, email <a href="mailto:marketplace@datadoghq.com">marketplace@datadoghq.com</a>. Once you have been granted access, you can review an [example pull request][12] in the Marketplace repository with annotations and best practices.

## Coordinate go-to-market (GTM) opportunities

Once a Marketplace tile is live, Technology Partners can meet with Datadog's Partner Marketing team to coordinate a joint go-to-market (GTM) strategy, which includes the following:

- A Datadog quote for partner press releases

> - A blog post on the [Datadog Monitor][23]

Reviewer 2: Broken link

Author: ```suggestion
- A blog post on the [Datadog Monitor][21]
```
